README.md

```diff
@@ -1,33 +1,26 @@
-# CVTemplate
+# Favourite Person
 
-Many professionals showcase their achievements through their online websites. Here are some exmaples:
-- https://stevenbartlett.com/
-- https://www.deborahmeaden.com/
+Many professionals showcase their achievements through their online websites. Here are some examples:
+
+- https://stevenbartlett.com/ (Portfolio website)
+- https://www.deborahmeaden.com/ (Portfolio website)
 - https://davidcthomas.co.uk/ (Portfolio website)
 - https://danielmoulton.me/ (HPE intern website)
 - https://kylethmas.github.io/ (My website)
-- https://ethanbrews.me
-- https://benchung.dev
-
-This is the starting template for this workshop: https://kylethmas.github.io/CVTemplate/
+- https://ethanbrews.me (HPE intern website)
+- https://benchung.dev (HPE intern website)
 
-## Starting From Scratch
+### Workshop Overview
 
-### Set up
-- Download visual studio code - https://code.visualstudio.com/Download
-- Download github desktop - https://desktop.github.com/
-- Create a github.com account
-- Fork and Clone this repository
+- We will be going through a PowerPoint presentation with instructions on how to complete each task. Then, we will be placing you in teams to work through the tasks, aiming to get as many done as you're able. Don't worry about trying to complete them all, just attempting the tasks is great! We will be running a live demonstration throughout, so hopefully you won't get too lost, but if you do please do reach out to any of our mentors who will be happy to help you out with any problems. Once the project has been complete, our mentors will be looking over all of them will decide a winning team. This will be based on our favourite design, so be creative with your solutions!
```

Comment:
```suggestion
- Using the PowerPoint presentation that covers the basic skills needed to solve each task and [W3Schools](https://www.w3schools.com/html/default.asp). 
- Aim to get as many tasks completed as possible. Don't worry about trying to complete them all, just attempting the tasks is great! 
- Your team mentor is here to help, if you have any questions ask them so hopefully you won't get too lost.
- Once the project has been completed, our judges will be looking over all of them and will decide a winning website. This will be based on the best user experience, aesthetically pleasing designs, and the inclusion of any cool features (found on W3Schools), so be creative with your solutions!